recipe_scrapers/rutgerbakt.py

```diff
@@ -0,0 +1,88 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+
+
+class RutgerBakt(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "rutgerbakt.nl"
+
+    def author(self):
+        return "Rutger van den Broek"
+
+    def title(self):
+        return self.schema.title().replace(" – recept", "").replace(" – Recept","")
+
+    def category(self):
+        category = self.url.split("rutgerbakt.nl/")[-1].split("/")[0].replace("-recepten", "").replace("-", " ")
+        return category
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        # The yields are all over the place. There is no way to parse this.
+        return 1
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def instructions(self):
+        # find the "instructions" heading It is not really clear when that is. So several steps need to be taken.
+        
+        # 1. Filter the headings.
+        # Some h2 headers contain photos. Those are not what we want.
+        def filterHeading(headings_old):
+            headings_new = []
+            for heading in headings_old:
+                if "class" in heading.attrs:
+                    if any(["photo" in attr.lower() for attr in heading.attrs["class"]]):
```

Comment:
```suggestion
                    if any("photo" in attr.lower() for attr in heading.attrs["class"]):
```

(please test this, but I think that it should be possible to convert this from a list comprehension into a generator comprehension)